rusk/src/lib/builder/node.rs

```diff
@@ -52,6 +52,9 @@ pub struct RuskNodeBuilder {
     command_revert: bool,
 }
 
+const DEFAULT_GAS_PER_DEPLOY_BYTE: u64 = 100;
```

Comment:
I'm very strongly in favor of increase the `DEFAULT_GAS_PER_DEPLOY_BYTE`, and getting rid of `DEFAULT_MIN_DEPLOYMENT_GAS_PRICE`. Not only does it make the code base simpler, it's also more "fair".

Reply:
That's a different topic, luckily this is something we can change without edit the contracts